app/src/main/java/com/infomaniak/mail/utils/AlertDialogUtils.kt

```diff
@@ -102,11 +102,11 @@ object AlertDialogUtils {
 
         MaterialAlertDialogBuilder(context)
             .setView(root)
-            .setPositiveButton(confirmButtonText, null)
+            .setPositiveButton(confirmButtonText) { _, _ -> onPositiveButtonClicked() }
```

Comment:
Won't this call `onPositiveButtonClicked()` twice?

Reply:
No because we override this clickListener in the  onShowListener